tests/test_db.py

```diff
@@ -238,3 +241,76 @@ def test_config_table(self):
         config = self._database.get_config_for(experiment_id)
 
         self.assertEqual(test_config, config)
+
+    # Test some metadata flow stuff and see if it is working.
+    def test_metadata(self):
```

Comment:
It looks like what this test does is creates a Config and configures a server using the same config string, and then verifies that the Config and the server database contain the same data. I would like to see a more explicit check that the database has correctly recorded `experiment_name = Lucas`, `metadata1 = one`, etc. There should also be checks to see what happens if metadata is malformed or missing (e.g., what happens if we don't supply `experiment_name`)? We should also probably add a test or two to the config tests to makes sure stuff is jsonifying correctly.

Reply:
[NOTE TO SELF FROM MAINTAINER MEETING]
originally check -> config object created but new code flips the result and can cause error. maybe include the code in there to check it?